output/batch/csv/formatter.go

```diff
@@ -0,0 +1,55 @@
+package csv
+
+import (
+	"encoding/csv"
+	"io"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/cube2222/octosql"
+	"github.com/cube2222/octosql/execution"
+	"github.com/cube2222/octosql/output/batch"
+)
+
+func TableFormatter(separator rune) batch.TableFormatter {
+	return func(w io.Writer, records []*execution.Record, watermark time.Time, errToPrint error) error {
+		var fields []string
```

Comment:
The function that I mentioned in batch/table/formatter.go should be used here (just a reminder)